libs/fundamental-redirects/index.js

```diff
@@ -1147,6 +1147,37 @@ for (const [pattern, path] of [
   );
 }
 
+function simpleRedirect(
+  pattern,
+  replacement,
+  { permanent = true, colonToSlash = false } = {}
+) {
+  return (path) => {
+    const match = pattern.exec(path);
+    if (match === null) {
+      return null;
+    }
+    const status = permanent ? 301 : 302;
+    let url = path.replace(pattern, replacement);
```

Comment:
Redirects return an absolute path but get a relative (with out leading `/`) so we have to prepend it here.

Reply:
I see my mistake now! 
```
▶ curl -I http://localhost:5000/en-US/docs/Core_JavaScript_1.5_Reference/Operators/Special_Operators/typeof_Operator
HTTP/1.1 301 Moved Permanently
Location: en-US/docs/Web/JavaScript/Reference/Operators/Special_Operators/typeof_Operator
```
That's clearly incorrect but because I didn't understand this, my tests were just wrong and fooled me. 

So, what change should I make to make this work?